.github/workflows/publish.yml

```
@@ -0,0 +1,30 @@
+name: Publish to PyPI
+
+on:
```

We could add a 
```suggestion
on:
  release:
    types: [published]
```

and also on `push` for testing from the PR